flink-connectors/flink-connector-filesystem/src/main/java/org/apache/flink/streaming/connectors/fs/StringWriter.java

```diff
@@ -82,7 +103,7 @@ public void open(FileSystem fs, Path path) throws IOException {
 	public void write(T element) throws IOException {
 		FSDataOutputStream outputStream = getStream();
 		outputStream.write(element.toString().getBytes(charset));
-		outputStream.write('\n');
+		outputStream.write(rowDelimiterBytes);
```

Comment:
You are still not testing anywhere, that `rowDelimiterBytes` field is being correctly used. Please add a test in `StringWriterTest` class, where you write some elements with standard and custom delimiter and that result is as you expected.

Reply:
Let me think about how to do it, thank you very much for your suggestion！

Reply:
@pnowojski  Hi , according to  your suggestions ,I add a test case!

Reply:
@pnowojski  Hello，do you have time to check it  for me?Thanks a lot !